openid-data-classes/src/commonMain/kotlin/at/asitplus/openid/SupportedCredentialFormatDefinition.kt

```diff
@@ -15,7 +15,7 @@ data class SupportedCredentialFormatDefinition(
      * according to (VC_DATA), Section 4.3, e.g. `VerifiableCredential`, `UniversityDegreeCredential`
      */
     @SerialName("type")
-    val types: Collection<String>? = null,
+    val types: Set<String>? = null,
```

Comment:
It just hit me: A set is unordered by definition. I know I voted for replacing lists with sets in the past and we made those changes, but… how do we guarantee serialization will be canonical? This is a larger issue. The easiest fix for this issue would be (globally) to use a sorted set, or a set that guarantees to preserve the order of elements. Could be that I am missing something here, but in case I am not, please assign an issue to me and I'll create a PR to fix this globally.

Reply:
I don't think there is ever an issue with canonicalization in serialization of OAuth structures ... but maybe I just don't see it?!

Reply:
I was more thinking about `SdJwtSigned` and other data that is serialized and sqigned. Signum might be affected too. I'm filing an issue and assigning you.